src/radiation/mocmc.cpp

```diff
@@ -277,22 +274,19 @@ TaskStatus MOCMCSampleBoundaries(T *rc) {
     species_d[s] = species[s];
   }
 
-  /// TODO: (LFR) Fix this junk
-  RadiationType dev_species[3] = {species[0], species[1], species[2]};
-
   auto swarm_d = swarm->GetDeviceContext();
 
   auto ix1_bc = rad->Param<MOCMCBoundaries>("ix1_bc");
-  auto ox1_bc = rad->Param<MOCMCBoundaries>("ox1_bc");
+  // auto ox1_bc = rad->Param<MOCMCBoundaries>("ox1_bc");
```

Comment:
clean this up?

Reply:
This needs to be properly implemented for all boundaries, I'll leave that for my next pr, `brryan/mocmc_torus` (I added a `TODO(BRR)` here)